filebeat/input/syslog/parser_test.go

```diff
@@ -31,6 +31,25 @@ func TestParseSyslog(t *testing.T) {
 		log    []byte
 		syslog event
 	}{
+		{
+			title: "Cisco's syslog",
+			log:   []byte("<190>589265: Feb 8 18:55:31.306: %SEC-11-IPACCESSLOGP: list 177 denied udp 10.0.0.1(53640) -> 10.100.0.1(15600), 1 packet"),
+			syslog: event{
+				priority:   190,
+				message:    "list 177 denied udp 10.0.0.1(53640) -> 10.100.0.1(15600), 1 packet",
+				hostname:   "%SEC-11-IPACCESSLOGP",
```

Comment:
Unfortunately this isn't a hostname. It's like a tag for the message type.

Reply:
I fixed it.